public/scripts/models/movieModel.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+var app = app || {};
+
+(function(module){
+  var movieModel = {
+    movieInputs: [],
```

Comment:
we need to discuss this to make sure we are using the same variable name across different files.